probe-rs/src/debug/language.rs

```diff
@@ -61,7 +54,19 @@ pub trait ProgrammingLanguage {
     }
 }
 
-#[derive(Clone)]
-pub struct UnknownLanguage;
+#[derive(Debug, Clone)]
+pub struct UnknownLanguage(DwLang);
 
-impl ProgrammingLanguage for UnknownLanguage {}
+impl ProgrammingLanguage for UnknownLanguage {
+    fn update_variable(
+        &self,
+        _variable: &Variable,
+        _memory: &mut dyn MemoryInterface,
+        _new_value: &str,
+    ) -> Result<(), DebugError> {
+        Err(DebugError::Other(anyhow::anyhow!(
+            "Updating variables for language {} is not supported.",
```

Comment:
We return `VariableValue::Empty` for reads, so would this even show up?

Reply:
Probably not, I've added an error message for the read as well.